graphql/schema.graphql

```diff
@@ -676,7 +676,7 @@ type Mutation {
   """
   Subscribe to newsletter
   """
-  subscribeNewsletter(email: String, isNewsletterOptIn: Boolean): Boolean
+  subscribeNewsletter(email: String, isNewsletterOptIn: Boolean, name: String): Boolean
```

Comment:
What do you think of accepting more fields, so we can solve the issue with the phone already?

```suggestion
  subscribeNewsletter(email: String, isNewsletterOptIn: Boolean, fields: NewsletterFields): Boolean
```

Reply:
But then how would we prevent anything in a user's profile from being overwritten?
Right now it's easy because we just need to check for `firstName`, but if we can set any field, we would need to validate each one of them

Reply:
Yes, that's it.

Reply:
Awesome! I'll add the phone field for now then.